hs/Svg/Parser.hs

```diff
@@ -43,17 +43,17 @@ performParse :: String -- ^ The title of the graph.
              -> String -- ^ The filename of the file that will be parsed.
              -> IO ()
 performParse graphName inputFilename =
-   do graphFile <- readFile (graphPath ++ inputFilename)
-      key <- insertGraph graphName
-      let parsedGraph = parseGraph key graphFile
-          PersistInt64 keyVal = toPersistValue key
-      print "Graph Parsed"
-      insertElements parsedGraph
-      print "Graph Inserted"
-      let genGraphPath = graphPath ++ "gen/"
-      createDirectoryIfMissing True genGraphPath
-      buildSVG key M.empty (genGraphPath ++ show keyVal ++ ".svg") False
-      print "Success"
+    do graphFile <- readFile (graphPath ++ inputFilename)
```

Comment:
Should the `do` keyword be on a new line? Right now it either ends a line or starts one.